src/components/Pages/Contact.vue

```diff
@@ -150,5 +150,19 @@ export default {
 }
 </script>
 
-<style>
+<style scoped>
+.page-heading {
+  /* preserve visual weight of previous h1 — adjust values to match your site's design if needed */
+  font-size: 28px;
+  font-weight: 500;
+  line-height: 1.2;
+  margin: 0 0 16px;
+}
+
+/* small-screen tweak to keep size reasonable on mobile */
+@media (max-width: 480px) {
```

Comment:
I'm not sure if this is a requirement UX has asked for, and I've not seen it elsewhere in the code base.